src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/StartFreeScanView.tsx

```diff
@@ -54,6 +58,11 @@ export function StartFreeScanView(props: Props) {
             <a
               href={process.env.NEXT_PUBLIC_HOW_IT_WORKS_SUMO_URL}
               target="_blank"
+              onClick={() => {
+                record("link", "click", {
+                  button_id: "returning_user_info_scan_learn_more",
```

Comment:
in figma, the property name is still `cta_id` fyi

Reply:
do we not need `page_location`?

Reply:
Thanks! `cta_id` might be more appropriate, but since changing it in the `metrics.yaml` comes with a review process we might stick to it for now.

Reply:
`link_id` might be more appropriate here, but since changing it in the `metrics.yaml` comes with a review process we might have to leave it as `button_id` for now.

Reply:
@mansaj Re `page_location`: We do append `path` to every telemetry event in `useTelemetry` [here](https://github.com/mozilla/blurts-server/blob/onboarding-instrumentration/src/app/hooks/useTelemetry.ts#L21).